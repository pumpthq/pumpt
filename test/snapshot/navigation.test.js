import React from 'react';
import Navigation from '../../src/components/main/navigation';
import renderer from 'react-test-renderer';

test('navigation renders properly', () => {
  const component = renderer.create(
    <Navigation type="onboarding">Nav Links</Navigation>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
