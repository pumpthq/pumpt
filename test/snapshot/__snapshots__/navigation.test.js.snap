// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigation renders properly 1`] = `
<nav
  className="navigation  navigation_type_onboarding"
>
  Nav Links
</nav>
`;
